common/istio/istio-install/components/ambient-mode/ztunnel.yaml

```diff
@@ -0,0 +1,177 @@
+---
+# Source: ztunnel/templates/rbac.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ztunnel
+  namespace: istio-system
+  labels:
+    app.kubernetes.io/name: ztunnel
+    app.kubernetes.io/managed-by: "Helm"
+    app.kubernetes.io/instance: "ztunnel"
+    app.kubernetes.io/part-of: "istio"
+    app.kubernetes.io/version: "1.27.1"
+    helm.sh/chart: ztunnel-1.27.1
+  annotations:
+    {}
+---
+# Source: ztunnel/templates/daemonset.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ztunnel
+  namespace: istio-system
```

Comment:
This looks like a duplication of common/istio/istio-install/components/ambient-mode/cni-ambient-patch.yaml.

Please make sure that there is no duplication in this PR.